### PR TITLE
Feature/calculator

### PR DESCRIPTION
**Pull request by jacksonjunelee**
added calculator to the sidenav

### CODE REVIEW

**Comment by victorhuberta** [comment]
As mentioned in our Telegram group, please change the UI design to fit what we already have. The style guide is coming soon. :+1: this is a good first pull request.

File: src/app/wallet/calculator/calculator.component.css [hunk 1]
```diff
@@ -7,19 +7,13 @@ h3 {
...
-  min-height: 1.6rem;
-  background-color: #FFF;
+.form-group {
+  text-align: center;
 }
 
-select {
-  height: 2rem;
-  width: 15.5rem;
+mat-form-field {
```
Comment (danjodanjo) [inline src/app/wallet/calculator/calculator.component.css]: Will update the class naming convention for each mat-form-field wrapped inputs. I accept this as a parent styling for every input for now

**Review by danjodanjo — APPROVED** [review]
Currently haven't decided naming class for each mat form field component. But good job for using the angular material!

File: src/app/wallet/calculator/calculator.component.css (hunk 1 above)
Comment (victorhuberta) [reply src/app/wallet/calculator/calculator.component.css]: I think with the view encapsulation provided by Angular for each component, the style won't affect other mat-form-fields.

**Comment by jacksonjunelee** [comment]
Yeah @danjodanjo. As @victorhuberta mentioned, only the calculator component will have those padding because of angular css encapsulation.

File: src/app/wallet/calculator/calculator.component.css (hunk 1 above)
Comment (jacksonjunelee) [reply src/app/wallet/calculator/calculator.component.css]: Yeah @danjodanjo. As @victorhuberta mentioned, only the calculator component will have those padding because of angular css encapsulation.

Comment (jacksonjunelee) [reply src/app/wallet/calculator/calculator.component.css]: I also think it'll be a good idea if we setup sass instead of using regular css as the project grows

**Comment by victorhuberta** [comment]
The styles are okay for now, after the style guide is finalized then we will update them again. But I tried the calculator with 1/1/2017 and it still calculates as if it is within the 3-years period. Could you please add a unit test that ensures anything before the ICO end date is zero? It's not as important in this context, but for semantic sake we should make it clear to the users.

Additionally, ICO refers to Initial Coin Offering, which in Exodia.World spans for 4 weeks. Right after ICO end date (the end of the 4 weeks), staking may start. So ICO_BETWEEN_DATE, ICO_AFTER_DATE, and ICO_BEFORE_DATE wrongly describe what their values are. If I understand your intention correctly, better names should be FIRST_INTEREST_PERIOD_END_DATE, SECOND_INTEREST_PERIOD_BETWEEN_DATE, and FIRST_INTEREST_PERIOD_BETWEEN_DATE. But they are a bit long.

**Comment by jacksonjunelee** [comment]
Usually I am a bigger fan of verbose. I can make that change
